### PR TITLE
checksum UI

### PR DESCRIPTION
**Pull request by cristianfchq**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 1]
```diff
@@ -23,6 +23,10 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: This blank line should be deleted

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 2]
```diff
@@ -80,43 +84,42 @@ public AudioConverterInterface() {
...
-                + "\nVolume: "
-                + settings.getVolume()
-                + "\nAudio Channel: "
-                + settings.getAudioChannel()
-                + "\nHz: "
-                + settings.getHz()
-                + "\nwith metadata: "
-                + settings.isMetadata());
+    public void actionPerformed(final ActionEvent e)  {
         try {
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:87]: This try-catch should have logs

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 3]
```diff
@@ -80,43 +84,42 @@ public AudioConverterInterface() {
...
         }
-        LOGGER.info("Finish");
     }
-
     /**
      * Obtains the request
      * @throws IOException
      */
     private void callRequest() throws IOException {
-        LOGGER.info("start");
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:114]: The start of the logger shouldn't be deleted

File: logs/propertieslogs.log [hunk 4]
```diff
@@ -954,3 +954,23 @@
...
+[INFO ] 2021-04-18 21:07:55.616 [AWT-EventQueue-0] AudioConverterInterface - Execute Try
+[INFO ] 2021-04-18 21:07:55.617 [AWT-EventQueue-0] ClientRequest - Start
+[INFO ] 2021-04-18 21:07:55.617 [AWT-EventQueue-0] ClientRequest - Execute Try
+[INFO ] 2021-04-18 21:07:55.617 [AWT-EventQueue-0] ClientRequest - Finish
+[INFO ] 2021-04-18 21:07:55.730 [http-nio-8080-exec-2] AudioConverterController - Execute Try
+[INFO ] 2021-04-18 21:07:55.752 [http-nio-8080-exec-2] AudioConverterController - finish
+[INFO ] 2021-04-18 21:07:55.753 [http-nio-8080-exec-2] RunCommand - start
+[INFO ] 2021-04-18 21:07:55.753 [http-nio-8080-exec-2] RunCommand - Execute Try
+[INFO ] 2021-04-18 21:07:56.064 [http-nio-8080-exec-2] RunCommand - finish
+[INFO ] 2021-04-18 21:08:01.859 [AWT-EventQueue-0] AudioConverterInterface - Finish
```
Comment (danielaSantaCruz) [inline logs/propertieslogs.log]: Logs should not be uploaded

File: logs/propertieslogs.log [hunk 5]
```diff
@@ -954,3 +954,23 @@
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - finish
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - start
 [INFO ] 2021-04-14 17:07:34.760 [http-nio-8080-exec-4] ExtractMetadata - Execute Try
+[INFO ] 2021-04-18 21:07:23.925 [main] ConverterApplication - start
```
Comment (edsonarios) [inline logs/propertieslogs.log]: The loguer should not go up

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 6]
```diff
@@ -23,6 +23,10 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+
+import static org.fundacion.jala.converter.service.ChecksumService.getFileChecksum;
+
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:28]: This blank line should be deleted

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 7]
```diff
@@ -80,43 +84,42 @@ public AudioConverterInterface() {
...
+                    + settings.getHz()
+                    + "\nwith metadata: "
+                    + settings.isMetadata()
+                    + "\nChecksum: "
+                    + getFileChecksum(file.getOriginFilePath()));
             callRequest();
-        } catch (Exception ex) {
-            LOGGER.error("Execute Exception to metaData conversion");
-            ex.printStackTrace();
+        } catch (IOException ioException) {
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:107]: This catch should have logs

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 8]
```diff
@@ -80,43 +84,42 @@ public AudioConverterInterface() {
...
+            ioException.printStackTrace();
+        } catch (NoSuchAlgorithmException noSuchAlgorithmException) {
+            noSuchAlgorithmException.printStackTrace();
```
Comment (edsonarios) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:111]: This catch should have logs

**Review by edsonarios — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java (hunk 3 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:114]: ok

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java (hunk 7 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:107]: ok

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java (hunk 8 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:111]: ok

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java (hunk 6 above)
Comment (cristianfchq) [reply src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java:28]: ok no problem

**Review by danielaSantaCruz — APPROVED** [review]

**Pull request merged**